js/src/util/bn.ts

```diff
@@ -90,17 +90,9 @@ function bignumToNumber<T extends BN<BigNumArray>>(bn: T) {
 }
 
 /** @ignore */
-export let bignumToString: { <T extends BN<BigNumArray>>(a: T): string };
+export const bignumToString = (<T extends BN<BigNumArray>>(a: T) => a.byteLength === 8 ? new a['BigIntArray'](a.buffer, a.byteOffset, 1)[0] : <any>decimalToString(a));
 /** @ignore */
-export let bignumToBigInt: { <T extends BN<BigNumArray>>(a: T): bigint };
-
-if (!BigIntAvailable) {
-    bignumToString = decimalToString;
-    bignumToBigInt = <any>bignumToString;
-} else {
-    bignumToBigInt = (<T extends BN<BigNumArray>>(a: T) => a.byteLength === 8 ? new a['BigIntArray'](a.buffer, a.byteOffset, 1)[0] : <any>decimalToString(a));
-    bignumToString = (<T extends BN<BigNumArray>>(a: T) => a.byteLength === 8 ? `${new a['BigIntArray'](a.buffer, a.byteOffset, 1)[0]}` : decimalToString(a));
-}
+export const bignumToBigInt = (<T extends BN<BigNumArray>>(a: T) => a.byteLength === 8 ? `${new a['BigIntArray'](a.buffer, a.byteOffset, 1)[0]}` : decimalToString(a));
```

Comment:
Seems like `bignumToString` can return a `bigint` here instead of a string. Are these implementations swapped?

Reply:
🤦 yes,I swapped them because the order of variables was different in the else.

Reply:
Fixed in 8ae01d7dd